### PR TITLE
Feat/Added Example 09 - trace mapping

### PR DESCRIPTION
**Pull request by rs-bh-n**
Added an Example 09 for tracemapping/Remote.

### CODE REVIEW

**Comment by rs-bh-n** [comment]
Hi @samigithub2022 , Please review this example.

**Comment by samigithub2022** [comment]
@koubaa - I didn't get a chance to review this. Last I checked, the documentation was running for 2 hours, I cancelled it. Can you please look into this?

@rs-bh-n

**Comment by rs-bh-n** [comment]
@samigithub2022  @koubaa 

Let me know if any code changes are needed from my side.

**Comment by dipinknair** [comment]
> 

Hi @rs-bh-n I am currently looking into this.

File: examples/basic/example_09_tracemapping.py [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+
+layers = imp_trace.Layers
+num_layers = layers.Count
+for layer in layers:
+    layer["Trace Material"] = "Copper Alloy"
+vias = imp_trace.Vias
+num_vias = vias.Count
+for via in vias:
+    via["Plating Material"] = "Copper Alloy"
+imp_trace.Import()
```
Comment (dipinknair) [inline examples/basic/example_09_tracemapping.py:209]: @koubaa , Bug fixed with imported load in 241, so keeping it open till release as you recommended

**Comment by dipinknair** [comment]
Trace mapping
image

**Review by sbradrick — APPROVED** [review]

**Pull request merged**